controllers/apps/opsrequest_controller.go

```diff
@@ -304,6 +306,27 @@ func (r *OpsRequestReconciler) parseAllOpsRequest(object client.Object) []reconc
 	return requests
 }
 
+func (r *OpsRequestReconciler) parseBackupOpsRequest(object client.Object) []reconcile.Request {
+	backup := object.(*dataprotectionv1alpha1.Backup)
+	var (
+		opsRequestSlice []appsv1alpha1.OpsRecorder
+		err             error
+		requests        []reconcile.Request
+	)
+	if opsRequestSlice, err = opsutil.GetOpsRequestSliceFromBackup(backup); err != nil {
+		return nil
```

Comment:
the relationship between backup and opsRequest is one-to-many?

Reply:
the relationship between backup is one-to-one.

Reply:
so the slice is not necessary, you can get the target OpsRequest from the backup's label